README.md

```diff
@@ -37,9 +37,8 @@ and use it if we need to.
 1. copy `.env.sample` to `.env` and fill in the non-production variables and remove the rest.
 1. make sure the variables in `.env` file is loaded to your shell, you can use [autoenv](https://github.com/inishchith/autoenv) for that
 1. review `config/database.yml` development section to make sure it connects to your postgres server
-1. run `rails db:setup`
+1. from root project directory run `docker-compose -f docker-compose-dev.yml up --build`
```

Comment:
[R] Instead of modifying these steps which works if you're developing directly on your machine, create a different section for docker development or mention both ways in this point

Reply:
very good, will make it